packages/create-platformatic-auto/src/for-marco.mjs

```diff
@@ -0,0 +1,118 @@
+import { say } from './say.mjs'
+import path, { basename } from 'node:path'
+import { safeMkdir } from './utils.mjs'
+import { createGitignore } from './create-gitignore.mjs'
+import { createGitRepository } from './create-git-repository.mjs'
+import { getPkgManager } from './get-pkg-manager.mjs'
+import pino from 'pino'
+import pretty from 'pino-pretty'
+import { execa } from 'execa'
+import ora from 'ora'
+import { createRequire } from 'node:module'
+import { pathToFileURL } from 'node:url'
+
+async function importOrLocal ({ pkgManager, name, projectDir, pkg }) {
```

Comment:
`name` seems unused

Reply:
yes it's unused. Will remove this file soon.